README.md

```diff
@@ -45,9 +45,228 @@ for example:
 4 - Put other slice on top of filling 🥪
```

Comment:
What is the  🥪 meant to be? Is it meant to be an emoji?

Reply:
It is meant to be a sandwich emoji 😂. I guess it is not clear then

Reply:
I think it's because I haven't updated my OS, my bad